modules/python/pylib/syslogng/__init__.py

```diff
@@ -0,0 +1,22 @@
+#!/usr/bin/env python
+#############################################################################
```

Comment:
@bazsi 
1. Some of the Python files contained this, some of them did not. I wanted to make it uniform.
2. I recommend using a shebang in each and every script to make file magic consistent on all platforms.

In light of these, do you agree?